build/archive.js

```diff
@@ -229,8 +229,13 @@ function exec(cmdLine, baton, callback) {
     }
 
     child_process.exec(cmdLine, function (err, stdout, stderr) {
-        var err = err || stderr;
-        err && baton ? error(err, baton) : callback((stdout || '').trim(), err);
+        if (!err) { err = stderr; }
+
+        if (err && baton) {
+            error(err, baton);
+        } else {
+            callback((stdout || '').trim(), err);
+        }
```

Comment:
@TimBarham The error for the or was (only in master):

    build/archive.js: line 232, col 17, 'err' is already defined.

Did you see that in master?